hooks/post-checkout

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
The hooks have been added automatically.
Is that part of `Sextant's` build process?

Reply:
This seems to be Git LFS-related stuff, you can safely remove this stuff to avoid including it into diff I suppose.

Reply:
Thanks for you participation in my PR!
I removed the files and added them to `.gitignore`. It seems to be a bug in git fort windows and VS.